Sources/ArcGISToolkit/Components/Augmented Reality/WorldScaleGeoTrackingSceneView.swift

```diff
@@ -137,21 +140,29 @@ public struct WorldScaleGeoTrackingSceneView: View {
         .task {
             do {
                 try await locationDataSource.start()
-                await withTaskGroup(of: Void.self) { group in
-                    group.addTask {
-                        for await location in locationDataSource.locations {
-                            lastLocationTimestamp = location.timestamp
-                            currentLocation = location
-                            await updateSceneView(for: location)
-                        }
-                    }
+                for await location in locationDataSource.locations {
+                    lastLocationTimestamp = location.timestamp
+                    currentLocation = location
+                    updateSceneView(for: location)
                 }
             } catch {}
         }
-        .toolbar {
-            ToolbarItem(placement: .bottomBar) {
-                if !calibrationViewIsHidden {
-                    calibrationView
+        .overlay(alignment: calibrationViewAlignment) {
+            if configuration is ARWorldTrackingConfiguration,
+               !calibrationViewIsHidden {
+                if !viewModel.isCalibrating {
+                    Button {
+                        viewModel.isCalibrating = true
+                        viewModel.setBasemapOpacity(0.5)
+                    } label: {
+                        Text("Calibrate")
+                    }
+                    .frame(maxWidth: .infinity)
```

Comment:
I like the previous small button, just my thoughts. We can later adjust the AR view safe area to avoid the bottom blank area.